src/server/services/country-codes/index.ts

```diff
@@ -0,0 +1,24 @@
+import { logger } from "../logger";
+import { countriesList } from "../../../server/services/metadata";
+
+export default function getCountryCodeFromCountryName(
+  countryName: string
+): string {
+  let selectedCountryCode = "";
+
+  if (!countryName) {
+    logger.error("getCountryCodeFromCountryName: Country name not found");
+  }
+
+  for (const countryData of countriesList) {
```

Comment:
could make this a little more concise by replacing the for loop with something like:

```suggestion
  selectedCountryCode = countriesList.find(country => {
    return country.text.toLowerCase() === countryName.toLowerCase();
  })?.code;
```

Reply:
Ah nice, thanks for this 👍